std_msgs/Bool trigger
------------------
float64 distance
